### ISSUE TITLE
User Story: RU_XX Product Sorting by Date added

### ISSUE THREAD

**Issue by DeeMcCart** [DeeMcCart/CI_PP5_Jeweller#37]
EPIC:  <epic>

As a **Returning User** I can **sort Products by date added** so that **I can see what products are new and interesting**

### Assumptions or Pre-Requisites:
* Product sorting already implemented Issue #32 
* Spare time at end of project as this is a nice-to-have

### Acceptance Criteria: (Must be completed before task is moved to 'Done')
- [x] Must be able to see newest products first
- [x] Ideally with a 'new' flag on the product

### Tasks

- [x] Task1 Ensure Products model has date_created field
- [x] Task2 Add js for sorting on Date-created ('Newest products first')
- [x] Task3 Add label to products that are < 10 days on display

Before changing task status to 'Review' or 'Done' please provide comment (and screenprints if appropriate) as documentary evidence of task completion

**Comment by DeeMcCart** [comment]
Sorting - delivered as standard.  Added check for product < 10 days old.


![Image](https://github.com/DeeMcCart/CI_PP5_Jeweller/assets/126502176/e5397253-bd08-43cd-a5b9-d6283175c61c)



![Image](https://github.com/DeeMcCart/CI_PP5_Jeweller/assets/126502176/ab98f8c1-7b99-4491-af74-b4b361017537)
